crypto/rand/rand_lib.c

```diff
@@ -25,6 +25,11 @@
 /* Macro to convert two thirty two bit values into a sixty four bit one */
 #define TWO32TO64(a, b) ((((uint64_t)(a)) << 32) + (b))
 
+/* Some libraries pretend to be glibc but don't support version checks */
+#ifndef __GLIBC_PREREQ
+#define __GLIBC_PREREQ(a, b) (0)
```

Comment:
Style nit, space before define.

Reply:
why should we define something from a reserved namescope at all?